Documentation/contributing/development/reviewers_committers/triage.rst

```diff
@@ -0,0 +1,153 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`cilium/cilium: ready to merge PRs
+<https://github.com/cilium/cilium/pulls?q=is%3Aopen+is%3Apr+label%3Aready-to-merge>`_
+
+Guidelines for merging:
+
+*   You should basically never be bypassing branch protections, even for
+    docs-only PRs. It's better to run ``/test`` regardless, and that should be
+    able to determine which tests are required. People should not be setting
+    ``ready-to-merge`` manually, we should always run ``/test``. If that runs
+    more CI than it should, that is a problem with CI.
+*   Once all required tests have passed, reviews are in, and discussions are
+    resolved, have a maintainer push the merge button without needing to bypass branch
+    protections. The button doesn't have to be green.
+*   `Maintainer's Little Helper
+    <https://github.com/cilium/maintainer-little-helper>`_ does not, at the
+    time of writing, know about the requirement to resolve discussions, so it
+    may flag PRs as ready-to-merge even when you can't click the merge button.
+    In this case, review the discussions and see if there are any outstanding
+    discussions, and if not, close them.
+
+PRs that may need attention
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Cilium community contributions (open, unassigned, not draft)
+<https://github.com/cilium/cilium/pulls?q=is%3Aopen+is%3Apr+no%3Aassignee+draft%3Afalse+label%3Akind%2Fcommunity-contribution+sort%3Aupdated-asc+-author%3Aapp%2Frenovate>`_
+
+
+*   Work your way forward from the oldest-updated PRs. PRs with recent
+    activity are likely being interacted with by other team members.
+*   Apply ``kind/*`` and ``release-note/*`` labels, external contributors
+    cannot apply labels.
+*   Approve workflows and run ``/test``.
+*   Re-run failed jobs due to flakes (e.g. connecting to 1.1.1.1, ..). Link or
+    create issues for persistent flakes.
+*   Bump the PR if feedback was positive but there's been no progress, either
+    from reviewer or author side.
+*   Kindly ask for rebases if conflicts emerge, or if e.g. things changed in
+    CI that are causing a job to fail.
+*   Make sure PRs contain no fixup or merge commits before merging them. Merge
+    PRs if the button is green. Do not override branch protection.
+*   If the contributor does not appear to be making progress on the PR, mark
+    the PR as draft and add a comment similar to:
+
+    *I've marked this PR as draft while you work through the feedback in the
+    PR above. When you're ready to continue with review & testing of the PR,
+    please click "Ready for Review" at the bottom of the page.*
+
+2. Triaging Bugs
+^^^^^^^^^^^^^^^^
+
+Triaging bugs means interacting with the reporter until the issue is clear and
+can be labeled with the corresponding vertical.
+
+The main goal of this activity is identifying crucial issues that could impact
+users, and gathering more info so that we can drive towards solutions to those
+issues. We want to identify serious bugs and fix them.
```

Comment:
This reads a bit weird - if a user filed an issue it's probably impacting users. Probably we can iterate a bit on the text to clarify this. I guess the goal here is more about establishing impact / severity.

Maybe it's time to formalize this in the docs and refer to it, I feel this categorization could help to guide triagers in this area:
https://github.com/cilium/cilium/issues/32269#issuecomment-2181736127